runtime/gc_vlhgc/CopyForwardScheme.hpp

```diff
@@ -203,15 +203,15 @@ class MM_CopyForwardScheme : public MM_BaseNonVirtual
 	 * @param cacheLock[in] The lock associated with the cache's owning list
 	 * @param wastedMemory[in] The number of bytes of memory which couldn't be used in the allocated portion of the copy cache and should be considered free memory when flushed back to the pool
 	 */
-	void discardRemainingCache(MM_EnvironmentVLHGC *env, MM_CopyScanCacheVLHGC *cache, MM_LightweightNonReentrantLock *cacheLock, UDATA wastedMemory);
+	void discardRemainingCache(MM_EnvironmentVLHGC *env, MM_CopyScanCacheVLHGC *cache, MM_LightweightNonReentrantLock *cacheLock, uintptr_t wastedMemory);
 	
 	/**
 	 * Determine whether the object pointer is found within the heap proper.
 	 * @return Boolean indicating if the object pointer is within the heap boundaries.
 	 */
-	MMINLINE bool isHeapObject(J9Object* objectPtr)
+	MMINLINE bool isHeapObject(J9Object *objectPtr)
 	{
-		return ((_heapBase <= (U_8 *)objectPtr) && (_heapTop > (U_8 *)objectPtr));
+		return ((_heapBase <= (uint8_t *)objectPtr) && (_heapTop > (uint8_t *)objectPtr));
```

Comment:
The outermost parentheses are specifically unwanted; the OMR [coding standard](https://github.com/eclipse-omr/omr/blob/master/doc/CodingStandard.md) says
> Do not enclose the return value in parentheses.